src/test/java/seedu/address/logic/parser/GroupCommandParserTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.GroupCommand;
+
+public class GroupCommandParserTest {
+    private GroupCommandParser parser = new GroupCommandParser();
+
+    @Test
+    public void allNamesAndTagValid() {
```

Comment:
Please follow the naming convention for test methods as follow:
![image](https://user-images.githubusercontent.com/70432640/140618255-659aac30-5bed-4a1a-974e-baeae13928cd.png)

This applies to your method below as well.